packages/web/index.js

```diff
@@ -143,14 +163,19 @@ module.exports = (neutrino, opts = {}) => {
         .use(optimize.ModuleConcatenationPlugin);
     })
     .when(neutrino.options.command === 'build', (config) => {
-      neutrino.use(clean, { paths: [neutrino.options.output] });
       neutrino.use(copy, {
         patterns: [{
           context: staticDir,
           from: '**/*',
           to: basename(staticDir)
         }]
       });
+
+      if (!options.html) {
```

Comment:
We already have a `.when` condition for `options.html`, so we can use that as well:

```js
.when(options.html
  (config) => {
    neutrino.use(htmlTemplate, options.htmlTemplate);
    config.plugin('script-ext').use(ScriptExtHtmlPlugin, [{ defaultAttribute: 'defer' }]);
  },
  (config) => config.plugin('manifest').use(ManifestPlugin)
)
```

Reply:
@eliperelman I had it there as to only include it when `neutrino.options.command === 'build'`